### DISCUSSION TITLE
Enable outer buttons on 6x3 crkbd

### DISCUSSION THREAD

**Discussion by beyertom** [manna-harbour/miryoku#104] [Q&A] [answered]
Hi all,

first of all, thank you for this awesome project!

I tried to activate the outer buttons on my 6x3 crkbd. I followed the guide posted [here](https://github.com/manna-harbour/miryoku/discussions/85) but i was not succesfull.

i tried these variations of custom_config.h:

```sh
#pragma once

#define XXX KC_NO

#define MIRYOKU_MAPPING( \
      K00,  K01,  K02,  K03,  K04,         K05,  K06,  K07,  K08,  K09, \
      K10,  K11,  K12,  K13,  K14,         K15,  K16,  K17,  K18,  K19, \
      K20,  K21,  K22,  K23,  K24,         K25,  K26,  K27,  K28,  K29, \
      N30,  N31,  K32,  K33,  K34,         K35,  K36,  K37,  N38,  N39 \
) \
LAYOUT_split_3x6_3( \
KC_A,  K00,  K01,  K02,  K03,  K04,         K05,  K06,  K07,  K08,  K09,  KC_D, \
KC_B,  K10,  K11,  K12,  K13,  K14,         K15,  K16,  K17,  K18,  K19,  KC_E, \
KC_C,  K20,  K21,  K22,  K23,  K24,         K25,  K26,  K27,  K28,  K29,  KC_F , \
                  K32,  K33,  K34,          K35,  K36,  K37 \
)
```

```sh
#pragma once

#define XXX KC_NO

#define MIRYOKU_MAPPING( \
KC_A,  K00,  K01,  K02,  K03,  K04,         K05,  K06,  K07,  K08,  K09,  KC_D, \
KC_B,  K10,  K11,  K12,  K13,  K14,         K15,  K16,  K17,  K18,  K19,  KC_E, \
KC_C,  K20,  K21,  K22,  K23,  K24,         K25,  K26,  K27,  K28,  K29,  KC_F , \
                  K32,  K33,  K34,          K35,  K36,  K37 \
)
```

i also tried to create the hex file with the FORCE_LAYOUT parameter:

`qmk compile -c -kb crkbd -km manna-harbour_miryoku -e FORCE_LAYOUT=split_3x6_3`
or
`make clean crkbd:manna-harbour_miryoku FORCE_LAYOUT=split_3x6_3`

flashing via:
`sudo avrdude -v -patmega32u4 -cavr109 -P/dev/ttyACM0 -b57600 -Uflash:w:"~/qmk_firmware/crkbd_rev1_manna-harbour_miryoku_split_3x6_3.hex":i`

Thank you very much for your support.

cheers

**Comment by beyertom** [comment, marked as answer]
i found a (workaround?) solution:

editing the config.h file in `~/qmk_firmware/layouts/community/split_3x6_3/manna-harbour_miryoku/` instead of using `custom_config.h`

**Reply by manna-harbour** [reply]
That is also fine, and necessary at the moment on `master`.

**Comment by manna-harbour** [comment]
If you are using the `master` branch, please see https://github.com/manna-harbour/miryoku/discussions/103#discussioncomment-3091616.

Your first variation here https://github.com/manna-harbour/miryoku/discussions/104#discussion-4192397 is the correct one.
